tests/testthat/test-joins.R

```diff
@@ -0,0 +1,58 @@
+actual_lpr_diag <- tibble::tibble(
+  recnum = rep(1:6, each = 2),
+  c_diag = 1:12,
+  c_diagtype = rep(c("A", "B"), 6)
+)
+
+actual_lpr_adm <- tibble::tibble(
+  pnr = rep(1:2, 3),
+  recnum = 2:7,
+  c_spec = 1:6,
+  d_inddto = c("20230101", "20220101", "20210101", "20200101", "20190101", "20180101"),
```

Comment:
Is `c_pattype` missing here?

Reply:
Um, `c_pattype` isn't in the variable list.